controllers/glance_controller.go

```diff
@@ -776,6 +771,15 @@ func (r *GlanceReconciler) generateServiceConfig(
 			glance.DatabaseName,
 		),
 	}
+	// We set in the main 00-config-default.conf the image-cache bits that will
+	// be used by CronJobs cleaner and pruner
+	if len(instance.Spec.ImageCacheSize) > 0 {
+		cacheSize := resource.MustParse(instance.Spec.ImageCacheSize)
```

Comment:
It looks like `resource.MustParse()` panics when something goes wrong [1].  Maybe we should catch the panic and report an error instead of allowing this to potentially crash the controller-manager pod here?

[1] https://github.com/kubernetes/apimachinery/blob/1ca6c696c8e470d0745788dad2f69de813b2237d/pkg/api/resource/quantity.go#L136-L142

Reply:
Makes sense and it doesn't hurt catching an issue here. Thanks for the hint. 
In theory if we reach this point it means that we already failed during the `pvc` creation, but I agree we should have full control of this block instead of relying in the general flow that could change over the time.
I'm going to update this part in my next iteration.

Reply:
ok I think we should survive to the `MustParse` panic there and produce proper log of the issue w/o crushing the Pod [1]
[1] https://go.dev/play/p/HKcBzeRsRXA